yaksh/documentation/moderator_docs/creating_quiz.rst

```diff
@@ -44,7 +46,9 @@ Creating a Exercise
 	.. image:: ../images/add_exercise.jpg
 
 	Exercise is similar to quiz with a difference that exercise has infinite attempts and 
-	exercise time depends on per question time. It also does not allow a student to skip the question.
+	infinite time. It also does not allow a student to skip the question.
+	Each question in an exercise can be timed i.e. time to solve a particular question.
+	Once the question time expires question solution is shown to the student.
```

Comment:
Once the question time expires, solution to the question is shown to the student.